Buildings/Fluid/Chillers/BaseClasses/PartialElectric.mo

```diff
@@ -378,21 +378,43 @@ for an example with a chiller operating in heating mode.
 </p>
 <h4>Implementation</h4>
 <p>
-Models that extend from this base class need to provide
-three functions to predict capacity and power consumption:
+This implementation computes the chiller capacity and power consumption
+the same way as documented in
+<a href=\"https://energyplus.net/assets/nrel_custom/pdfs/pdfs_v22.1.0/EngineeringReference.pdf\">
+EnergyPlus v22.1.0 Engineering Reference</a>
+section 14.3.9.2.
+The chiller capacity is computed as
+</p>
+<pre>  QEva_flow_ava = QEva_flow_nominal*capFunT</pre>
+<p>
+and the power consumption is computed as
+</p>
+<pre>  P = -QEva_flow_ava/COP_nominal*EIRFunT*EIRFunPLR*CR.</pre>
```

Comment:
Can you please also add the definition of these functions. Best may be in `Buildings.Fluid.Chillers.Data.ElectricEIR.Generic`
While referencing the EnergyPlus documentation is fine, we like to have the models to be fully documented and avoid that users need to consult other software documentation to understand the models.

Reply:
To clarify, do you mean adding the documentation of these two formulae to `Data.ElectricEIR.Generic` as well?

Reply:
Addressed in cf4d6ec. I added a table detailing how these functions are formulated.

Reply:
I revised the documentations further in cb19747.
I moved the table to `Buildings.Fluid.Chillers.BaseClasses.PartialElectric` which I feel is closer to a "center of traffic" that makes sense to provide information across models. Using `capFunT` as an example, now in the same document it is easy to find both "what it is" (i.e. `QEva_flow_ava = QEva_flow_nominal * capFunT` up-text) and "how it is computed" (in the table) close to each other.
Please let me know if this addressed your concern.